tyr/servers/server.py

```diff
@@ -806,8 +812,12 @@ def connection(self):
 
         while True:
             try:
-                connection.connect(self.instance.public_dns_name,
-                                    username = 'ec2-user')
+                keys = ['~/.ssh/stage', '~/.ssh/prod']
```

Comment:
It may work with `ssh-agent` on OS X even if the `key_filename` argument is used, but I haven't tested it. When the keys are added to the agent on Amazon Linux, it doesn't see them and requires that I pass in the key locations. Even if it does still check the agent, it would throw an error when those files don't exist.